test/helpers/kubectl.go

```diff
@@ -110,6 +110,7 @@ func GetCurrentIntegration() string {
 type Kubectl struct {
 	Executor
 	*serviceCache
+	BasePath string
```

Comment:
Since we have `.BasePath()` on the SSHMeta type, maybe it would be easier to add that to Executor and use that where we use this field?